### PR TITLE
add card , update scripts, write playwrite test

### PR DESCRIPTION
**Pull request by v2i0s2h2**
the test is take screen shot of web page and store in poject dir

### CODE REVIEW

File: scripts/deploy-local.sh [hunk 1]
```diff
@@ -23,9 +23,3 @@ sleep 5
 # Generate declarations
 echo "Generating declarations..."
 dfx generate backend
-
-# Start the development server
-echo "Starting development server..."
-npm run dev
-
-echo "Deployment process completed successfully!"
```
Comment (Web3NL) [inline scripts/deploy-local.sh:31]: After succesfull deploy of frontend and backend, you could display this message

**Review by Web3NL — COMMENTED** [review]
You added `playwright-test.sh`. Usually a command like that would be in `package.json`

**Review by Web3NL — COMMENTED** [review]
its not clear how to run the test. put the command in package.json and make a readme on how to bootstrap the project locally and run the test.

**Review by Web3NL — COMMENTED** [review]
its not clear how to run the test. put the command in package.json and make a readme on how to bootstrap the project locally and run the test. 
(I had to do npx playwright install first)

**Review by Web3NL — COMMENTED** [review]
Only keep the form and card on the main page. Use gix for styling the form.

**Comment by v2i0s2h2** [comment]
> Only keep the form and card on the main page. Use gix for styling the form.

sir you want to delete whoami function

File: scripts/deploy-local.sh (hunk 1 above)
Comment (v2i0s2h2) [reply scripts/deploy-local.sh:31]: written readme on how to bootstrap the project locally and run the test.
remove test script and add on package.json

File: README.md [hunk 2]
```diff
@@ -1,38 +1,71 @@
...
+2. Install dependencies:
+   ```
+   npm install
+   ```
 
-```bash
-npm run dev
+3. Install playwright:
+   ```
+   npx playwright install
```
Comment (Web3NL) [inline README.md:27]:
```suggestion
   npm playwright install --save-dev
```

File: README.md [hunk 3]
```diff
@@ -1,38 +1,71 @@
...
+### Running the application locally
+
+1. Start the Internet Computer local network:
+   ```
+   dfx start --clean
+   ```
+
+2. In a new terminal, make the deploy script executable and run the development server:
+   ```
+   chmod +x ./scripts/deploy-local.sh
```
Comment (Web3NL) [inline README.md:40]: Not needed!

File: README.md [hunk 4]
```diff
@@ -1,38 +1,71 @@
...
 
-> To deploy your app, you may need to install an [adapter](https://kit.svelte.dev/docs/adapters) for your target environment.
+- [Svelte](https://svelte.dev/) - The web framework used
+- [Internet Computer](https://internetcomputer.org/) - Deployment platform
+- [Playwright](https://playwright.dev/) - Testing framework
+
+## License
+
+This project is licensed under the MIT License - see the [LICENSE.md](LICENSE.md) file for details
+```
```
Comment (Web3NL) [inline README.md:71]:
```suggestion
```

**Review by Web3NL — COMMENTED** [review]
Made some small comments on the readme file. Fix them and merge this. Then we continue with a new PR

**Comment by v2i0s2h2** [comment]
edit readme

**Pull request merged**